packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -507,9 +507,9 @@ const disableVmHighAvailability = async (xapi: Xapi, vm: Vm) => {
 // │  │  ├─ task.start(message: 'transfer')
 // │  │  │  ├─ task.warning(message: string)
 // │  │  │  └─ task.end(result: { size: number })
-// │  │  ├─ task.start(message: 'merge')
+// │  │  ├─ task.start(message: type === full ? 'clean' : 'merge')
 // │  │  │  ├─ task.warning(message: string)
-// │  │  │  └─ task.end(result: { size: number })
+// │  │  │  └─ task.end(result?: { size: number })
```

Comment:
Not bad but I think it's preferable to split these tasks to correctly specify their result.